snippets/snippets.service.js

```diff
@@ -4,6 +4,20 @@ const snippetsService = {
   getAll: () => {
     const snippets = snippetsRepository.findAll();
     return snippets;
+  },
+  create: (snippetData) => {
+
+    const title = snippetData.title;
+    const content = snippetData.content;
+    const category = snippetData.category;
+    const author =snippetData.author;
+
+    if(title || content || category || author) {
```

Comment:
Can you implement a [guard clause](https://medium.com/@timothydan/javascript-guard-clauses-64b999e3240) [1])(https://www.geeksforgeeks.org/guard-clause-in-javascript/) here? Instead of doing a if/esle condition?